src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -17608,6 +17665,192 @@ public Test2()
             VerifyModelForOutVar(model, x1Decl, x1Ref);
         }
 
+        [Fact]
+        public void ConstructorInitializers_06()
```

Comment:
>ConstructorInitializers_06 [](start = 20, length = 26)

What coverage is this intended to add above what is covered by 1Scope_ConstructorInitializers_03` and `ConstructorInitializers_01`?

Reply:
> What coverage is this intended to add above what is covered by 1Scope_ConstructorInitializers_03andConstructorInitializers_01`?

This covers references to the locals within constructors body, as opposed to references in the initializer itself

---
In reply to: [170056838](https://github.com/dotnet/roslyn/pull/24918#discussion_r170056838) [](ancestors = 170056838)

Reply:
`Scope_ConstructorInitializers_03` covers references to the locals within the constructor's body, though I see that `ConstructorInitializers_06` declares it as an argument of the ctor-initializer rather than a nested invocation.

Reply:
> Scope_ConstructorInitializers_03 covers references to the locals within the constructor's body, though I see that ConstructorInitializers_06 declares it as an argument of the ctor-initializer rather than a nested invocation.

Yes, you correct that is another difference.

---
In reply to: [170119727](https://github.com/dotnet/roslyn/pull/24918#discussion_r170119727) [](ancestors = 170119727)